app/wrappers/wakes/facebook_metrics_wrapper.rb

```diff
@@ -22,7 +22,7 @@ def share_counts
           hash[r['id']] = r.fetch('share').fetch('share_count')
         rescue KeyError
           # Some urls can have no 'share' keys. This is generally in the case when the url isn't valid.
-          Rails.logger.warn "Share count not received for #{r['id']}. Possibly an invalid url?"
+          Rails.logger.warn "Share count not received for #{r['id']}. Response is: #{r}"
```

Comment:
Oh, and I don't mean to keep that log that way for good. May be we can improve what gets logged once we learn what is happening? (`id` being `nil` seems strange). 

Not directly related, but we are running an older version of the api. We should probably create a task to have it be upgraded at an appropriate time.